specification/peering/resource-manager/readme.cli.md

```diff
@@ -8,8 +8,13 @@ cli:
   cmd-override:
     "^.*[/]peerasns[/].*$": "peering asn"
     "^.*[/]peerings[/].*$": "peering"
+    "^.*[/]legacypeerings$": "peering legacy"
+    "^.*[/]peeringlocations$": "peering locations"
+    "^.*[/]peeringservicelocations$": "peering service location"
```

Comment:
Could it be peering service list-location?

Reply:
will change in next round